### PR TITLE
chore: remake vday update

### PR DESCRIPTION
**Pull request by treagan82**

### CODE REVIEW

File: _posts/2018-02-14-weekly-update.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Alan is putting the moves on the website this week as he continues to style it in its entirety.  Changes will
+start being approved and implemented any time now.
+
+Todd is still building content, Meet the team content has been added!  Not all of the info posted is correct,
+but the team is trying to squeeze in a bio while furiously coding everything for this sprint so be sure to check
+back periodically for updates.  The Development Tools and Software page is in the works as well, so be on the
+lookout for updates there as well.
+
+Matt is still proving to be a gitjitsu master, he working to sharpen the skills of the rest of team.  Another
```
Comment (cclinard84) [inline _posts/2018-02-14-weekly-update.md]: > he working to sharpen the skills of the rest of team.

Change to

> he's working to sharpen the skills of the rest of team.

File: _posts/2018-02-14-weekly-update.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+week or so he could earn the rank of gitjitsu grand master!  Doing code reviews and managing work flow are also
+on the docket for him this week.  He will be taking the wheel on the Demo Nick has been working on, to add a
```
Comment (cclinard84) [inline _posts/2018-02-14-weekly-update.md]: > He will be taking the wheel on the Demo Nick has been working on,

needs to read 
> He will also be taking the wheel on the Demo Nick has been working on,

File: _posts/2018-02-14-weekly-update.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+few sliders and filters
+
+Bryan has been churning out code like butter all week long, and will be continuing to pump it out for the next
+few days, until he shifts gears to bug fixing for a few days.  Once that is complete he will be back on the gas
+petal for coding.
```
Comment (cclinard84) [inline _posts/2018-02-14-weekly-update.md]: Flowers have petals, cars have pedals 😃

File: _posts/2018-02-14-weekly-update.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Nick has finished his demo project and will be working on filters but should be finished by the end of this
+sprint.  He is also going to be assisting Alan with the design for the website.
```
Comment (cclinard84) [inline _posts/2018-02-14-weekly-update.md]: > Nick has finished his demo project and will be working on filters but should be finished by the end of this
  sprint.  He is also going to be assisting Alan with the design for the website.

Lets say: 
> Nick has finished the initial version of the demo project and will now be helping Alan with creating the website theme. He will also be reviewing Matt's Pull Requests for a change!

Comment (treagan82) [reply _posts/2018-02-14-weekly-update.md]: I will make all these changes and resubmit.  Im thinking I should probably start doing the typing in word haha.

**Pull request merged**